app/components/pay-page/choose-membership-plan-modal/invoice-details-step.hbs

```diff
@@ -0,0 +1,45 @@
+<div>
+  <div class="relative">
+    <PayPage::ChooseMembershipPlanModal::PlanCard
+      @isSelected={{true}}
+      @onClick={{(noop)}}
+      @plan={{@selectedPlan}}
+      @promotionalDiscount={{if (eq @selectedPlan.id "yearly") @activeDiscountForYearlyPlan null}}
+      @regionalDiscount={{@regionalDiscount}}
+    />
+
+    <div class="absolute inset-0 bg-white/60 flex items-center justify-center opacity-100 transition-opacity duration-200">
```

Comment:
```suggestion
    <div class="absolute inset-0 bg-white/60 flex items-center justify-center">
```

The design preview initially used animations, but we aren't using those anymore and instead use a 60% opacity background (`bg-white/60`)